src/save_failed_screen.c

```diff
@@ -399,3 +399,47 @@ static bool8 WipeSectors(u32 sectorBits)
     else
         return TRUE;
 }
+
+void CB2_FlashNotDetectedScreen(void)
```

Comment:
This entire functions uses tabs instead of spaces (should be spaces)

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5188/commits/ef5c47af5e9c2deeddee50650086f3f8076ae475